test/remote/gateways/remote_ipg_test.rb

```diff
@@ -3,9 +3,9 @@
 class RemoteIpgTest < Test::Unit::TestCase
   def setup
     @gateway = IpgGateway.new(fixtures(:ipg))
-
+    @gateway_ma = IpgGateway.new(fixtures(:ipg_ma).merge({ store_id: nil }))
     @amount = 100
-    @credit_card = credit_card('5165850000000008', brand: 'mastercard', verification_value: '987', month: '12', year: '2029')
+    @credit_card = credit_card('5165850000000008', brand: 'mastercard', month: '12', year: '2029')
```

Comment:
removed the verification_value because their api kept responding "CVV checks not supported on this card"